src/components/settingsModal.html

```diff
@@ -36,6 +36,7 @@ <h5 class="modal-title">Settings</h5>
                                 <tr data-bind="template: { name: 'MultipleChoiceSettingTemplate', data: Settings.getSetting('eggAnimation')}"></tr>
                                 <tr data-bind="template: { name: 'MultipleChoiceSettingTemplate', data: Settings.getSetting('hideHatchery')}"></tr>
                                 <tr data-bind="template: { name: 'MultipleChoiceSettingTemplate', data: Settings.getSetting('farmDisplay')}"></tr>
+                                <tr data-bind="template: { name: 'BooleanSettingTemplate', data: Settings.getSetting('breedingEfficiency+')}"></tr>
```

Comment:
I think we rather want this in the hatchery window, as it's something players might often want to turn on and off while looking there

Reply:
I considered this, and decided against it for two reasons.
1. I didn't want to bother with the HTML
2. The changes also affect the Pokémon List

I could see adding an additional option in the hatchery, or just moving it there entirely. I'd have to see how easy/difficult the html is

Reply:
Added the setting, it looks bad because I'm bad at HTML. Feel free to fix or reject